plugins/services/src/js/stores/__tests__/MarathonStore-test.js

```diff
@@ -138,6 +138,9 @@ describe("MarathonStore", function() {
       var marathonApps = MarathonStore.get("apps");
 
       Object.keys(marathonApps).forEach(function(key) {
+        if (!marathonApps[key].health) {
```

Comment:
This is needed, because otherwise the test breaks because of images in marathonApps.

Reply:
Could you please elaborate? What test is breaking? What is the expectation?

Reply:
Here https://github.com/dcos/dcos-ui/pull/3820/files#diff-7b6a93d2ba53835d3ed69d3f2e183f45R483 we add images property to Applications and this is a property which didn't exist when this test was written, so the test expected images to have a key and failed.

Reply:
Do you mean that the old fixture is incomplete?

Reply:
The `forEach` definitely is.

Reply:
As I can see in this foreach we just check every marathon app from the store. I fail to understand why would we ever want start skipping some of them?

Reply:
Here we are checking for health checks, so we don't care about marathon apps with no health, so we skip them. If you insist, I can also try changing the fixtures.

Reply:
It's not me insisting, I just don't see where we add new fixtures? So in my understanding this test shouldn't fail. Other question would be: how exactly doest it fail?

Reply:
We don't add new fixtures, but we add the possibility for Applications to have images. It fails with `can't find key of undefined` on this line - https://github.com/dcos/dcos-ui/pull/3820/files/e2567b8eedc40c48808e924003772b32ff30ebe6#diff-4da2835a0dd453e39652befc964aa197R150 if I remember correctly.

Reply:
I understand why the test fails if it meets this condition, but I am unclear why this PR's changes would have caused this to fail? Why is `health` affected by the images changes?